aspnet/hosting/servicing.rst

```diff
@@ -0,0 +1,34 @@
+.. _hosting-servicing:
+
+Servicing
+=========
+
+By `Sourabh Shirhatti`
+
+ASP.NET 5 supports servicing of runtime components (DNX) and packages through Windows Update Services which will deliver updates to patch any vulnerabilities if and when they are discovered. This document provides an overview of how to setup your Windows Server correctly to receive updates.
```

Comment:
Nit: "Update Services which will deliver" to "Update Services, which will deliver" (add comma)